doc/contributor/interop.md

```diff
@@ -386,6 +386,10 @@ They have priority over methods that the foreign object actually provides.
 
 `object.name` sends `INVOKE`.
 
+`object + other` for will send `IS_BOXED` on `object` and based on that will
```

Comment:
`for will`?
There should be mention this is done for all operators, as in the CHANGELOG message.

Reply:
Fixing.